ivy/functional/frontends/numpy/mathematical_functions/rounding.py

```diff
@@ -0,0 +1,20 @@
+import ivy
+
+
+def ceil(
+    x,
+    /,
+    out=None,
+    *,
+    where=True,
+    casting="same_kind",
+    order="k",
+    dtype=None,
+    subok=True,
```

Comment:
I suppose that this function also takes two extra optional keyword only arguments: signature, extobj, and you should recreate this behaviour.